lib/perl/Genome/Model/ClinSeq/Command/AnnotateGenesByCategory.t

```diff
@@ -77,13 +86,13 @@ ok(-e $log_file2, "Wrote message file from annotate-genes-by-category to a log f
 #Perform a diff between the stored results and those generated by this test
 my @diff = `diff -r -x '*.log.txt' -x '*.pdf' -x '*.stderr' -x '*.stdout' $expected_output_dir $temp_dir`;
 ok(@diff == 0, "Found only expected number of differences between expected results and test results")
-or do {
-  diag("expected: $expected_output_dir\nactual: $temp_dir\n");
-  diag("differences are:");
-  diag(@diff);
-  my $diff_line_count = scalar(@diff);
-  print "\n\nFound $diff_line_count differing lines\n\n";
-  Genome::Sys->shellcmd(cmd => "rm -fr /tmp/last-annotate-genes-by-category");
-  Genome::Sys->shellcmd(cmd => "mv $temp_dir /tmp/last-annotate-genes-by-category");
-};
+    or do {
+    diag("expected: $expected_output_dir\nactual: $temp_dir\n");
+    diag("differences are:");
+    diag(@diff);
+    my $diff_line_count = scalar(@diff);
+    print "\n\nFound $diff_line_count differing lines\n\n";
+    Genome::Sys->shellcmd(cmd => "rm -fr /tmp/last-annotate-genes-by-category");
+    Genome::Sys->shellcmd(cmd => "mv $temp_dir /tmp/last-annotate-genes-by-category");
+    };
```

Comment:
I wonder why perltidy has the contents of this block and the closing `}` at the same indentation.